src/components/explore/ProbeExplorer.svelte

```diff
@@ -150,9 +155,14 @@
   // client counts.
   const MULT = 1.1;
   $: clientCountsData = clientCounts(data);
+  $: sampleCountsData = sampleCounts(data);
+  $: yValsSample = sampleCountsData.map((d) => d.totalSample);
   $: yVals = clientCountsData.map((d) => d.totalClients);
-  $: yMax = Math.max(50, Math.max(...yVals));
-  $: yClientsDomain = [0, yMax * MULT];
+  $: yMaxClient = Math.max(50, Math.max(...yVals));
```

Comment:
Can we add a `const` for this `50` instead of leaving it as a [Magic Number](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)?

Reply:
this comment is a dupe! IDK what happened.